exotic/utils.py

```diff
@@ -99,6 +98,22 @@ def round_to_2(*args):
 
 # Credit: Kalee Tock
 def get_val(hdr, ks):
+    """
+    Pluck the value for a certain key from myriad possible known keys
+
+    Parameters
+    ----------
+    hdr : dict
+        a dictionary of observatory data
+    ks : list[str]
+        a list of possible keys? PR FIXME: not sure really
```

Comment:
This `get_val` function is only called from the `find` function in the same file. The arguments, `hdr` and `ks` are first passed into the `find` function and then passed into this `get_val` function.

`find` is called from `inputs.py` in the `elevation`, `latitude` and `longitude` functions. (see lines 396, 334 and 364 respectively.) The caller in the `latitude` function is looks like this:

```python
lat = find(hdr, ['LATITUDE', 'LAT', 'SITELAT'])
```
Clearly it's looking for a latitude and I'm guessing that different equipment manufacturers output some known expected values for latitude.

Knowing the expected values of `hdr` and `ks`, and where they originate, would help me to provide better method documentation and help anyone else that works on this later add to this method and make it more precise. Any historical perspective would be greatly appreciated.

Reply:
@kaleetock this is really helpful to a noob like me!

I was struggling to understand that `hdr` meant "FITS Header" but that totally makes sense now in the context of your explanation.

The PR has been updated based on your comments. I think it will help future developers of this project build up their knowledge of how all this works.

Your explanation gives me a good idea to help make the code more clear. I'll open an issue to explain it more detail. Thanks!